include/glow/Backends/BackendUtils.h

```diff
@@ -21,6 +21,43 @@
 #include "glow/IR/IR.h"
 
 namespace glow {
+namespace runtime {
+/// RuntimeSymbolInfo
+/// Contains information for initialization and handling of symbol at runtime.
+struct RuntimeSymbolInfo {
+  /// The size in bytes.
+  size_t size;
+  /// Offset in bytes from the base address.
+  size_t offset;
+  /// Type of symbol.
+  Type type;
+};
+/// Runtime Bundle
```

Comment:
Same nit (name of thing in comment).

Reply:
I'd also prefer a blank line between top-level class/struct declarations.  (I dunno if this is in the official coding standards but we do it most other places.) (In general I hate blank lines but this is my one exception.) (I also love parenthetical comments.)